src/TextToTalk/TextToTalk.cs

```diff
@@ -391,6 +402,21 @@ private void BackendSay(SayRequest request)
             this.backendManager.Say(request);
         }
 
+        private string GetSpeakerRace(GameObject? speaker)
+        {
+            var race = this.data.GetExcelSheet<Race>();
+            if (race is null || speaker is null || speaker is not Character ch)
```

Comment:
`speaker is null` can be removed from this, the `speaker is not Character ch` handles that already.